lms/static/js/edxnotes/views/toggle_notes_factory.js

```diff
@@ -0,0 +1,73 @@
+;(function (define, undefined) {
+'use strict';
+define([
+    'jquery', 'underscore', 'backbone', 'gettext', 'js/edxnotes/views/visibility_decorator'
+], function($, _, Backbone, gettext, EdxnotesVisibilityDecorator) {
+    var ToggleNotesView = Backbone.View.extend({
+        events: {
+            'click .action-toggle-notes': 'toogleHandler'
+        },
+
+        errorMessage: gettext('Cannot save your state. This may be happening because of an error with our server or your internet connection. Try refreshing the page or making sure you are online.'),
```

Comment:
@srpearce 

Changed the error message to:
```We couldn't <show/hide> your notes. Make sure that you are connected to the Internet, and then try refreshing the page.``` Just to be sure, the ```< >``` around ```show/hide``` are intentional?

Also, did the same change to the error message in ```toggle_notes_factory_spec.js```